MSAL/src/native_auth/network/responses/validator/sign_up/MSALNativeAuthSignUpResponseValidator.swift

```diff
@@ -172,7 +178,8 @@ final class MSALNativeAuthSignUpResponseValidator: MSALNativeAuthSignUpResponseV
 
     private func handleContinueError(_ error: Error, with context: MSIDRequestContext) -> MSALNativeAuthSignUpContinueValidatedResponse {
         guard let apiError = error as? MSALNativeAuthSignUpContinueResponseError else {
-            return .unexpectedError
+            MSALLogger.log(level: .error, context: context, format: "Sign Up Continue: Error was not decoded properly, error: \(error)")
```

Comment:
Here and in other I've added the "Sign Up Continue" and in other places I've specified on which Endpoint/Flow the failure happens to have them all in sync.